src/main/java/org/eclipse/tractusx/ssi/lib/exception/json/TransformJsonLdException.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * ******************************************************************************
+ * Copyright (c) 2021,2023 Contributors to the Eclipse Foundation
```

Comment:
Please update all headers to `Copyright (c) 2021,2024 Contributors to the Eclipse Foundation` on the files you've touched.